src/i18n/en.js

```diff
@@ -403,7 +392,7 @@ export default {
     error_phone_too_short: `Ouch! The phone number is too short.`,
     error_register_denied: `Hold on! You've reached the maximum accounts allowed.`,
     error_required_field: `Ouch! The field is empty.`,
-    error_unexpected_HTML: `Unexpected error. Please try again* or [contact Support](${mailtoSupport}).
+    error_unexpected_HTML_MD: `Unexpected error. Please try again* or [contact Support](${mailtoSupport}).
```

Comment:
Why here and in other places you have double suffix? (`_HTML_MD`)

Reply:
i change now and only has _MD